.github/workflows/ci.yml

```diff
@@ -139,6 +139,14 @@ jobs:
             ./gradlew :app:record${{ env.flavour }}${{ env.config }}AndroidTestScreenshotTest -Pandroid.testInstrumentationRunnerArguments.variant=${{ matrix.variant }}
             ./scripts/check-no-changes.sh
 
+      - name: Upload Screenshot Tests Artifact
```

Comment:
this will help us look at the screenshot test outputs in general

Reply:
This is a great idea 👍